src/exports.jl

```diff
@@ -737,6 +738,8 @@ export inverse
 export invert
 export invert_birational_map
 export inverted_set
+export invlex
+export revlex # to be removed
```

Comment:
This export list is meant to be sorted alphabetically. And anyway, I think `@deprecate` exports the old name by default, so this line should just not be needed, I think...
```suggestion
```